Calibration/IsolatedParticles/plugins/IsolatedTracksCone.cc

```diff
@@ -1,48 +1,323 @@
+  std::vector<std::vector<int> >*     t_v_hsimInfoConeNRest;
+  std::vector<std::vector<int> >*     t_v_hsimInfoConeNPhoton;
+  std::vector<std::vector<int> >*     t_v_hsimInfoConeNNeutHad;
+  std::vector<std::vector<int> >*     t_v_hsimInfoConeNCharHad;
+  std::vector<std::vector<int> >*     t_v_hsimInfoConeNTotal;
+
+  std::vector<std::vector<double> >*  t_v_hsimCone;
+  std::vector<std::vector<double> >*  t_v_hCone;
+  std::vector<std::vector<int> >*     t_v_nRecHitsCone;
+  std::vector<std::vector<int> >*     t_v_nSimHitsCone;
+
+  std::vector<std::vector<double> >*  t_v_distFromHotCell;
+  std::vector<std::vector<int> >*     t_v_ietaFromHotCell;
+  std::vector<std::vector<int> >*     t_v_iphiFromHotCell;
+
+  std::vector<std::vector<int> >*     t_v_hlTriggers;
+  std::vector<int>*                   t_hltHB;
+  std::vector<int>*                   t_hltHE;
+  std::vector<int>*                   t_hltL1Jet15;
+  std::vector<int>*                   t_hltJet30;
+  std::vector<int>*                   t_hltJet50;
+  std::vector<int>*                   t_hltJet80;
+  std::vector<int>*                   t_hltJet110;
+  std::vector<int>*                   t_hltJet140;
+  std::vector<int>*                   t_hltJet180;
+  std::vector<int>*                   t_hltL1SingleEG5;
+  std::vector<int>*                   t_hltZeroBias;
+  std::vector<int>*                   t_hltMinBiasHcal;
+  std::vector<int>*                   t_hltMinBiasEcal;
+  std::vector<int>*                   t_hltMinBiasPixel;
+  std::vector<int>*                   t_hltSingleIsoTau30_Trk5;
+  std::vector<int>*                   t_hltDoubleLooseIsoTau15_Trk5;
+
+  std::vector<std::vector<int> >*     t_v_RH_h3x3_ieta;
+  std::vector<std::vector<int> >*     t_v_RH_h3x3_iphi;
+  std::vector<std::vector<double> >*  t_v_RH_h3x3_ene ;
+  std::vector<std::vector<int> >*     t_v_RH_h5x5_ieta;
+  std::vector<std::vector<int> >*     t_v_RH_h5x5_iphi;
+  std::vector<std::vector<double> >*  t_v_RH_h5x5_ene ;
+  std::vector<std::vector<int> >*     t_v_RH_r26_ieta ;
+  std::vector<std::vector<int> >*     t_v_RH_r26_iphi ;
+  std::vector<std::vector<double> >*  t_v_RH_r26_ene  ;
+  std::vector<std::vector<int> >*     t_v_RH_r44_ieta ;
+  std::vector<std::vector<int> >*     t_v_RH_r44_iphi ;
+  std::vector<std::vector<double> >*  t_v_RH_r44_ene  ;
+
+  std::vector<unsigned int>*          t_irun;
+  std::vector<unsigned int>*          t_ievt;
+  std::vector<unsigned int>*          t_ilum;
+  
+
+};
+
 IsolatedTracksCone::IsolatedTracksCone(const edm::ParameterSet& iConfig) :
    trackerHitAssociatorConfig_(consumesCollector()) {
 
   //now do what ever initialization is needed
-  doMC            = iConfig.getUntrackedParameter<bool>  ("DoMC", false); 
-  myverbose_      = iConfig.getUntrackedParameter<int>( "Verbosity", 5 );
+  doMC_            = iConfig.getUntrackedParameter<bool>  ("DoMC", false); 
+  myverbose_      = iConfig.getUntrackedParameter<int>( "Verbosity", 5);
```

Comment:
please use lower case for the first letter of python configuration parameter names